pyproject.toml

```diff
@@ -1,39 +1,54 @@
-[tool.mypy]
-python_version = 3.9
-namespace_packages = true
-incremental = false
-cache_dir = ""
-warn_redundant_casts = true
-warn_return_any = true
-warn_unused_configs = true
-warn_unused_ignores = false
-allow_redefinition = true
-disallow_untyped_calls = false
-disallow_untyped_defs = true
-disallow_incomplete_defs = true
-check_untyped_defs = true
-disallow_untyped_decorators = false
-strict_optional = true
-strict_equality = true
-explicit_package_bases = true
-follow_imports = "skip"
-ignore_missing_imports = true
```

Comment:
It looks like we are removing quite a few mypy args. Is there a reason for this?

Reply:
Ye a lot of them were either default, or very strict and not the default. I kept only the super necessary ones :smile: